pywinauto/controls/uia_controls.py

```diff
@@ -1302,6 +1307,84 @@ def check_button(self, button_identifier, make_checked, exact=True):
         self.actions.logSectionEnd()
         return button
 
+    # -----------------------------------------------------------
+    def items(self):
+        return self.buttons()
+
+    # -----------------------------------------------------------
+    def _activate(self, item, is_last):
+        """Activate the specified item"""
+        if not item.is_active():
+            item.set_focus()
+        try:
+            item.expand()
+        except(NoPatternInterfaceError):
```

Comment:
This is different catching style than everywhere in the project. Use space and no rounds brackets.